src/components/interactive-image/interactive-image.scss

```diff
@@ -0,0 +1,33 @@
+@import "../common/_base";
+
+@mixin size($width, $height) {
+  height: $height;
+  width: $width;
+}
+
+gx-interactive-image {
+  /**
+  * @prop --control-height:
+   * Set the height or the control. *Must be set before render the control*
+   * (auto by default)
+  */
+  --control-height: false;
+
+  /**
+	* @prop --control-width:
+	 * Set the width or the control. *Must be set before render the control*
+	 * (100% by default)
+	*/
+  --control-width: false;
+
+  @include visibility(flex);
+  @include size(var(--control-width), var(--control-height));
+  background-position: center;
+  background-repeat: no-repeat;
+  overflow: hidden;
+
+  background-color: rgba(250, 0, 0, 0.2);
```

Comment:
Why are we settings this default `background-color`?

Reply:
You're right.

Reply:
fixed in da6f90c